.github/workflows/ci.yaml

```
@@ -76,20 +43,28 @@ jobs:
+        uses: actions/checkout@v5.0.0
+        with:
+          persist-credentials: false
 
       - name: Move python script that replaces private package registry information to temp folder so it doesn't get deleted
         run: |
           mv .github/workflows/replace_private_package_registries.py $RUNNER_TEMP
 
-      - name: Install python tooling
-        uses: ./.github/actions/install_deps_uv
+      - name: Install tooling
+        uses: ./.github/actions/install_deps
```

actionlint (1.7.9)

could not parse action metadata in "/home/jailuser/git/.github/actions/install_deps": line 4: unexpected key "type" for definition of input "skip-updating-devcontainer-hash"

(action)